File: app/frontend/store/rootReducer.ts
import { combineReducers } from '@reduxjs/toolkit';

// Example slice for testing
const dummyReducer = (state = {}, action: { type: string }) => {
  switch (action.type) {
    default:
      return state;
  }
};

const rootReducer = combineReducers({
  dummy: dummyReducer,
});

export type RootState = ReturnType<typeof rootReducer>;
export default rootReducer; 

File: backend/app/app/frontend/store/rootReducer.py
from typing import Any, Dict


const dummyReducer = (state = {}, action: Dict[str, Any]) => {
  switch (action.type) {
    default:
      return state
  }
}
const rootReducer = combineReducers({
  dummy: dummyReducer,
})
RootState = ReturnType[typeof rootReducer]
default rootReducer 